tests/rust/test_rust_backend.py

```
@@ -0,0 +1,674 @@
+                    results.append((thread_id, result))
+            except Exception as e:
+                errors.append((thread_id, e))
+
+        threads = []
+        for i in range(3):
+            thread = threading.Thread(target=compile_in_thread, args=(i,))
+            threads.append(thread)
+            thread.start()
+        for thread in threads:
+            thread.join(timeout=30)
+        assert len(results) <= 3
```

Ruff (0.11.9)

Magic value used in comparison, consider replacing `3` with a constant variable

(PLR2004)